src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -235,17 +237,22 @@ export const NavBar: React.FC = track(
                   Artworks
                 </NavItem>
 
-                <NavItem href="/auctions">Auctions</NavItem>
-                <NavItem href="/viewing-rooms">Viewing&nbsp;Rooms</NavItem>
-                <NavItem href="/galleries">Galleries</NavItem>
-                <NavItem href="/fairs">Fairs</NavItem>
-                <NavItem href="/Shows">Shows</NavItem>
-                <NavItem
-                  // Hide link at smaller viewports — corresponding display inside of `MoreNavMenu`
-                  // If we need to do this again, consider a more abstract solution
-                  display={["none", "none", "flex", "flex"]}
```

Comment:
I think we can reduce the changes to this PR to just removing this `display` line. 

(If not familiar with this pattern, our design system Palette uses ["responsive props" via styled-system](https://styled-system.com/responsive-styles)